dotCMS/src/main/java/com/dotcms/util/user/LiferayUserTransformer.java

```diff
@@ -0,0 +1,76 @@
+package com.dotcms.util.user;
+
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.liferay.portal.ejb.UserPool;
+import com.liferay.portal.model.User;
+import java.util.Map;
+
+/**
+ * This class implement the UserTransformer interface
+ * to convert dotconnect maps from user to User
+ */
+public class LiferayUserTransformer implements UserTransformer {
+
+
+    @Override
+    public User fromMap(Map<String, Object> map) throws DotStateException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)